autogen/agentchat/conversable_agent.py

```diff
@@ -321,6 +322,75 @@ def reply_func(
         if ignore_async_in_sync_chat and inspect.iscoroutinefunction(reply_func):
             self._ignore_async_func_in_sync_chat_list.append(reply_func)
 
+    @staticmethod
+    def simple_nested_chat(chat_queue, recipient, messages, sender, config):
+        """A simple chat reply function.
+        This function initiate one or a sequence of chats between the "recipient" and the agents in the
+        chat_queue.
+
+        It extracts and returns a summary from the nested chat based on the "summary_method" in each chat in chat_queue.
+        """
+        last_msg = messages[-1].get("content", "")
```

Comment:
Empty string is scary, because some models might not support empty string.

How about changing the default string to "NA" or "Empty message"?

Reply:
Thanks, addressed!